src/Strategy/Python/Setuptools.hs

```diff
@@ -42,14 +42,19 @@ findProjects = walk' $ \dir _ files -> do
     ([], Nothing) -> pure ([], WalkContinue)
     _ -> pure ([project], WalkContinue)
 
-getDeps :: (Has ReadFS sig m, Has Diagnostics sig m) => SetuptoolsProject -> m (Graphing Dependency, GraphBreadth)
+getDeps :: (Has ReadFS sig m, Has Diagnostics sig m) => SetuptoolsProject -> m DependencyResults
 getDeps project = do
   graph <-
     context "Setuptools" $
       Diag.combineSuccessful
         "Analysis failed for all requirements.txt/setup.py in the project"
         [analyzeReqTxts project, analyzeSetupPy project]
-  pure (graph, Partial)
+  pure $
+    DependencyResults
+      { dependencyGraph = graph
+      , dependencyGraphBreadth = Partial
+      , dependencyManifestFiles = (maybe [] pure (setuptoolsSetupPy project)) ++ setuptoolsReqTxt project
```

Comment:
[nit] this is also known as `maybe [] pure` is also known as `Data.Maybe.maybeToList`
```hs
      , dependencyManifestFiles = maybeToList (setuptoolsSetupPy project) ++ setuptoolsReqTxt project
```

Reply:
Oh amazing, that's good to know